src/util.js

```diff
@@ -114,11 +86,9 @@ export const rejectionOf = promise =>
     reason => reason
   );
 
-export let config;
 export let xo;
 beforeAll(async () => {
   console.log("beforeAll");
-  config = await getConfig();
```

Comment:
why move it to a dedicated `beforeAll`?

Reply:
To separate concerns, it could easily be moved in it's own file.